docs/ADD_NATIVE_OFFERS_TO_MARKETPLACE.md

```diff
@@ -0,0 +1,108 @@
+### Adding a Custom Earn or Spend Offers to the Kin Marketplace Offer Wall ###
+
+The Kin Marketplace offer wall displays built-in offers, which are served by the Kin Ecosystem Server. Their purpose is to provide users with opportunities to earn initial Kin funding, which they can later spend on spend offers provided by hosting apps.
+
+You can also choose to display a banner for your custom offer in the Kin Marketplace offer wall. This serves as additional "real estate" in which to let the user know about custom offers within your app. When the user clicks on your custom Spend or Earn offer in the Kin Marketplace, your app is notified, and then it continues to manage the offer activity in its own UX flow.
+
```

Comment:
line4: your app will be notified, and then you can continue to manage the offer within your app. ??